contrib/completion/zsh/_docker-compose

```diff
@@ -359,10 +362,17 @@ _docker-compose() {
 
     _arguments -C \
         '(- :)'{-h,--help}'[Get help]' \
-        '--verbose[Show more output]' \
-        '(- :)'{-v,--version}'[Print version and exit]' \
         '(-f --file)'{-f,--file}'[Specify an alternate docker-compose file (default: docker-compose.yml)]:file:_files -g "*.yml"' \
         '(-p --project-name)'{-p,--project-name}'[Specify an alternate project name (default: directory name)]:project name:' \
+        '--verbose[Show more output]' \
+        '(- :)'{-v,--version}'[Print version and exit]' \
+        '(-H --host)'{-H,--host}'[Daemon socket to connect to]:host:' \
```

Comment:
Yeah, I'll break the commit up.

Regarding the sorting, I reordered the flags to match the order given by `docker-compose help x`. I figured this would make the script easier to maintain.

Do you want me to do alphabetical ordering instead?